runtime/gc_base/GCExtensions.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2020 IBM Corp. and others
+ * Copyright (c) 1991, 2021 IBM Corp. and others
```

Comment:
This file after all has not been changed. Would you please just exclude it from change set without modification of copyrights data?

Reply:
Done!